packages/app/src/cli/services/dev/select-store.ts

```diff
@@ -44,7 +35,8 @@ export async function selectStore(
   // Then, with a store selected, make sure its transfer-disabled, prompting to convert if needed
   let store = await selectStorePrompt(stores, showDomainOnPrompt)
   if (!store) {
-    outputInfo(`\n${await CreateStoreLink(org.id)}`)
+    const createStoreLink = await developerPlatformClient.getCreateDevStoreLink(org.id)
+    outputInfo(createStoreLink)
```

Comment:
using `renderInfo` puts everything in `info` boxes, which I'm unsure about the UX of it. nonetheless I've changed it because I think it's good to keep things consistent, but I'll see what UX feedback is for this :)